src/ChatList.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="chat-list" :style="{background: colors.chatList.bg}">
+    <table>
+      <tbody>
+        <tr v-for="chat in chatList" :key="chat.id" @click="openChat(chat.id)">
```

Comment:
I would mark this `tr` to have a `cursor: pointer;` so that the user understands that is clickable :)